mobile/pages/Login.js

```diff
@@ -1,24 +1,184 @@
-import { StyleSheet, Button, Text, View } from 'react-native';
+import * as React from 'react';
+import { 
+    StyleSheet, 
+    Button, 
+    Text, 
+    View, 
+    TouchableOpacity, 
+    Image, 
+    TextInput
+} from 'react-native';
+
+import {useForm, Controller} from 'react-hook-form';
+import axios from "axios";
+import { useReducer } from "react";
+import { useNavigation } from '@react-navigation/native';
+
+    
+const Login = ()  => { 
+
+    const navigation = useNavigation();
+
+    const {setValue, handleSubmit, errors, control} = useForm({
+        defaultValues: {
+            'email': '',
+            'password': ''
+        }
+    });
+
+    const [state, setState] = useReducer((state, newState) => ({ ...state, ...newState }), {
+        forgotPassword: false,
+        alert: "",
+    });
+
+    const loggedIn = null;
+    const onPressLogin = (data) => {
+        console.log(data.email);
+        console.log(data.password);
+        
+        axios
+            .post("https://pwdly.herokuapp.com/api/user/login", {  
+                email: data.email,
+                password: data.password,
+            })
+            .then((res) => {
+                if (res.status === 200) { 
+                    navigation.navigate('Dashboard')
```

Comment:
"res.data.user.token is our JWT token, we need to store it. Use secure store: https://docs.expo.dev/versions/latest/sdk/securestore/ or async-storage: https://docs.expo.dev/versions/latest/sdk/async-storage/ to save it" 

I think you can just copy Adam's implementation from Sign up